### PR TITLE
Feature/pool 379 make the prize start time configurable

### PR DESCRIPTION
**Pull request by robsecord**

### CODE REVIEW

File: contracts/prize-strategy/PrizeStrategy.sol [hunk 1]
```diff
@@ -95,6 +96,9 @@ contract PrizeStrategy is PrizeStrategyStorage,
 
     prizePeriodSeconds = _prizePeriodSeconds;
     prizePeriodStartedAt = _currentTime();
+    if (_prizePeriodStart > prizePeriodStartedAt) {
```
Comment (asselstine) [inline contracts/prize-strategy/PrizeStrategy.sol:99]: Let's keep it simple and just set the prize period start to the passed start *always*

File: contracts/prize-strategy/PrizeStrategy.sol [hunk 2]
```diff
@@ -852,12 +866,15 @@ contract PrizeStrategy is PrizeStrategyStorage,
   }
 
   modifier requireCanStartAward() {
+    require(_isPrizePeriodStarted(), "PrizeStrategy/prize-period-not-started");
```
Comment (asselstine) [inline contracts/prize-strategy/PrizeStrategy.sol:869]: ✂️ 

We already have isPrizePeriodOver()

File: contracts/prize-strategy/PrizeStrategy.sol [hunk 3]
```diff
@@ -852,12 +866,15 @@ contract PrizeStrategy is PrizeStrategyStorage,
...
     require(_isPrizePeriodOver(), "PrizeStrategy/prize-period-not-over");
     require(!isRngRequested(), "PrizeStrategy/rng-already-requested");
     _;
   }
 
   modifier requireCanCompleteAward() {
+    require(_isPrizePeriodStarted(), "PrizeStrategy/prize-period-not-started");
```
Comment (asselstine) [inline contracts/prize-strategy/PrizeStrategy.sol:876]: ✂️ 

We already have isPrizePeriodOver

File: contracts/prize-strategy/PrizeStrategy.sol (hunk 1 above)
Comment (robsecord) [reply contracts/prize-strategy/PrizeStrategy.sol]: done, tests updated

File: contracts/prize-strategy/PrizeStrategy.sol (hunk 2 above)
Comment (robsecord) [reply contracts/prize-strategy/PrizeStrategy.sol]: done

File: contracts/prize-strategy/PrizeStrategy.sol (hunk 3 above)
Comment (robsecord) [reply contracts/prize-strategy/PrizeStrategy.sol]: done

**Pull request merged**